client/my-sites/stats/stats-comments/index.jsx

```diff
@@ -21,7 +21,10 @@ export default React.createClass( {
 	displayName: 'StatsComments',
 
 	propTypes: {
-		site: PropTypes.object,
+		site: PropTypes.oneOfType( [
+			PropTypes.object,
+			PropTypes.bool
```

Comment:
I will be working a bit more in that component when I move comments over to redux - so I'll tidy that up further then.